src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.jdbc.dao;
+
```

Comment:
in this class for all methods u catch creation of Connection class, but u already did it on ConnectionUtil.getConnection()